src/Module/Contact.php

```diff
@@ -646,21 +646,25 @@ public static function content(array $parameters = [], $update = 0)
 			return $arr['output'];
 		}
 
+		$select_uid = $_SESSION['uid'];
```

Comment:
Isn't this supposed to be `local_user()`?

Reply:
I just copied it just from some lines below, always after the principle of "never changing a running system". :-)